### PR TITLE
Tech leader

### PR DESCRIPTION
**Pull request by ManuelLopezAnido**
Routes done ✓
Navbar component done ✓
Layout component done ✓

### CODE REVIEW

**Review by leimeterjoaquin — APPROVED** [review]

**Review by MicaQuarleri — APPROVED** [review]

File: src/Components/Postulants/Form/index.js [hunk 1]
```diff
@@ -321,7 +321,7 @@ const PostulantsForm = () => {
       body: JSON.stringify(data)
     })
       .then((response) => response.json())
```
Comment (aquilesmariani) [inline src/Components/Postulants/Form/index.js:323]: If you're not using the response,  I think it's better not to parse the result and just redirect the page will be fine

File: src/Components/Positions/Form/index.js [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+  };
+  const onChangeDatePosted = (event) => {
+    setDatePosted(event.target.value);
+  };
+  const onChangeClosingDate = (event) => {
+    setClosingDate(event.target.value);
+  };
+
+  const params = new URLSearchParams(window.location.search);
+  const PosId = params.get('id');
```
Comment (aquilesmariani) [inline src/Components/Positions/Form/index.js:40]:
```suggestion
  const posId = params.get('id');
```

File: src/Components/Positions/Form/index.js [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+    }
+
+    fetch(url, options)
+      .then((response) => {
+        if (response.status !== 200) {
+          return response.json().then(({ ErrMessage }) => {
+            throw new Error(ErrMessage);
+          });
+        }
+        return response.json();
```
Comment (aquilesmariani) [inline src/Components/Positions/Form/index.js:101]: Let's redirect here instead of using a new `.then`

File: src/Components/Header/index.js [hunk 4]
```diff
@@ -1,65 +1,38 @@
 import styles from './header.module.css';
+import { useLocation } from 'react-router-dom';
 
 function Header() {
+  const location = useLocation();
+  let Entity = location.pathname;
```
Comment (aquilesmariani) [inline src/Components/Header/index.js:6]:
```suggestion
  let entity = location.pathname;
```

**Review by aquilesmariani — CHANGES_REQUESTED** [review]
Just a few comments to go

**Review by aquilesmariani — APPROVED** [review]

**Pull request merged**